server/sonar-db-dao/src/main/java/org/sonar/db/webhook/WebhookDeliveryDao.java

```diff
@@ -55,4 +79,10 @@ public void deleteComponentBeforeDate(DbSession dbSession, String componentUuid,
   private static WebhookDeliveryMapper mapper(DbSession dbSession) {
     return dbSession.getMapper(WebhookDeliveryMapper.class);
   }
+
+  public Map<String, WebhookDeliveryLiteDto> selectLatestDeliveries(DbSession dbSession, List<WebhookDto> webhooks) {
```

Comment:
Minor - it would be better to define public methods before private ones (before `mapper()` in the current case).